### DISCUSSION TITLE
How can i get GitHub edc. pack without a picture?

### DISCUSSION THREAD

**Discussion by scffs** [community/community#39764] [GitHub Education] [answered]
My camera is actually pretty bad so how can I upload screenshot or anything else to proof my academic status?

**Comment by david-engelmann** [comment]
I'm not sure there is a way to avoid the "Use a Camera" option in the UI. I would submit your problem [detailing your camera issue here](https://support.github.com/contact/education). You can also check out the ["What proof can I provide to support my application?" section on this page](https://support.github.com/request/education)!

**Reply by scffs** [reply]
i'll try

**Reply by david-engelmann** [reply]
Don't forget to "Mark as Answer"

**Reply by scffs** [reply]
This is not answer
My problem still unresolved

**Reply by david-engelmann** [reply]
I'm not sure there is anything further I can do to assist

**Reply by scffs** [reply]
Ok, thank you anyway
I'll try to talk w/ github support

**Reply by david-engelmann** [reply]
Good Luck!

**Comment by samus-aran** [comment, marked as answer]
Hi @scffs 👋 

Have you tried applying via a smart phone (if you have one?) and use the 'desktop mode' on the phones browser. That would hopefully provide a better camera option.

**Reply by scffs** [reply]
I don't think so.
I also need to translate the docs and to legibly see the date but it is handwritten and cannot be applied by the GitHub system

**Reply by samus-aran** [reply]
If you wanted to re-apply using a photo then I would suggest using a Dated Student ID. If it is non English then get a piece of paper and translate it best you can into English. Lay it down side by side your Student ID and using your phone with a camera (if you have one) take a photo that includes the ID + translated copy in one shot. Ensure that your proof contains the name of your school and your own name (we compare your GitHub profile to assist us) at least 👍 

Let us know how you get on.

**Reply by scffs** [reply]
Dude thank u so much, I think I can get the status by phone cam

**Reply by scffs** [reply]
Have done 
Thx I appreciate it much

**Reply by samus-aran** [reply]
Awesome stuff @scffs 🎉 Glad I could help 👍
